kolibri/plugins/learn/assets/src/views/explore-page/index.vue

```diff
@@ -13,19 +13,7 @@
       {{ topic.description }}
     </p>
 
-    <content-card-grid :contents="contents" v-if="contents.length">
-
-      <template scope="content">
-        <content-card
-          :key="content.id"
-          :title="content.title"
-          :thumbnail="content.thumbnail"
-          :kind="content.kind"
-          :progress="content.progress"
-          :link="genLink(content)"/>
-      </template>
-
-    </content-card-grid>
+    <content-card-grid :contents="contents" :gen-link="genLink" v-if="contents.length" />
```

Comment:
Delegating the card rendering to the content grid definitely seems more appropriate.